app/views/games/show.html.erb

```diff
@@ -1,2 +1,92 @@
-<h3>Show Page</h3>
-<%= @game.name %>
\ No newline at end of file
+<div class="main">
+  <div class="left">Filler</div>
+  <div class="board-container">
+
+    <div class="board">
+       <div class="row" id="row-8">
+          <div class="square" id="0-7"><%= render_piece(0,7) %></div>
```

Comment:
I may have missed this in other pull requests (sorry if I did!) but does `render_piece` display the piece from the specific game with the x, y coordinates that are passed as parameters? I don't have any suggestions for changes here really, more a general question to understand what's happening here.

Reply:
Yes, `render_piece` looks up the `piece` at the the location given in the arguments. For example, `render_piece(0, 7)` will find the piece with `position_x: 0` and `position_y: 7` (which is a rook), and then display the corresponding ASCII character ♜ based on `piece.name`.